deploy/charts/ceph-csi-drivers/templates/serviceaccount.yaml

```diff
@@ -8,11 +8,19 @@ kind: ServiceAccount
 metadata:
   name: {{ $normalizedDriverName }}-ctrlplugin-sa
   namespace: {{ $root.Release.Namespace }}
+{{- with $root.Values.imagePullSecrets }}
```

Comment:
`{{- with $root.Values.imagePullSecrets }}` should be `{{- with .Values.imagePullSecrets }}` because we have root at the higher level